### PR TITLE
Branch add picture to applicant

### PR DESCRIPTION
**Pull request by lihaoquan**

### CODE REVIEW

**Review by YuvBindal — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/EditApplicantCommand.java [hunk 1]
```diff
@@ -103,7 +103,8 @@ public CommandResult execute(Model model) throws CommandException {
         } else {
             applicantToEdit = new Applicant((Person) personToEdit,
                     editApplicantDescriptor.getRole().orElse(new Role(DEFAULT_ROLE)),
-                    editApplicantDescriptor.getStage().orElse(new Stage(DEFAULT_STAGE)));
+                    editApplicantDescriptor.getStage().orElse(new Stage(DEFAULT_STAGE)),
+                    editApplicantDescriptor.getImg().orElse(""));
```
Comment (xinen26) [inline src/main/java/seedu/address/logic/commands/EditApplicantCommand.java:107]: Looks good to me! Just to check, would EditApplicantCommandParser and EditCommandParser need to be edited to reflect the changes?

**Review by xinen26 — COMMENTED** [review]
LGTM but just have a question I wanted to ask

File: src/main/java/seedu/address/logic/commands/EditApplicantCommand.java (hunk 1 above)
Comment (lihaoquan) [reply src/main/java/seedu/address/logic/commands/EditApplicantCommand.java:107]: Yes, one of the test cases can't pass if it doesn't have this

**Pull request merged**